doc/governance.rst

```diff
@@ -21,14 +21,56 @@ Bodies
 Ceph Executive Council
 ======================
 
-Responsibilities
-----------------
+.. _exec-council-responsibilities:
+
+Ceph Executive Council Responsibilities
+---------------------------------------
+
+- Spokesperson
+
+   - welcome/keynote for cephalocon
```

Comment:
Suggest capitalizing the bullets, and of course Cephalocon

Reply:
https://github.com/ceph/ceph/pull/60142